docker/services/cron/crontab

```diff
@@ -25,8 +25,8 @@ MAILTO=""
 2 2 * * 1 root /usr/local/bin/python /code/listenbrainz/manage.py spark request_missing_mb_data >> /logs/stats.log 2>&1
 # Calculate user similarity
 3 2 * * * root /usr/local/bin/python /code/listenbrainz/manage.py spark cron_request_similar_users >> /logs/stats.log 2>&1
-4 2 * * * root /usr/local/bin/python /code/listenbrainz/manage.py spark request_troi_playlists --slug weekly-jams >> /logs/troi_spark.log
-5 2 * * * root /usr/local/bin/python /code/listenbrainz/manage.py spark request_troi_playlists --slug weekly-exploration >> /logs/troi_spark.log
+4 2 * * 0 root /usr/local/bin/python /code/listenbrainz/manage.py spark request_troi_playlists --slug weekly-jams >> /logs/troi_spark.log
```

Comment:
I wonder if we should hold off on making this change -- now we need to ensure that all the new code for sending to spotify works, and doing it on a daily basis for now will let us solve this much faster

Reply:
Reverted.